Stripe/PublicHeaders/STPAPIClient.h

```diff
@@ -271,6 +274,13 @@ static NSString *const STPSDKVersion = @"15.0.0";
                                                    country:(NSString *)countryCode
                                                   currency:(NSString *)currencyCode;
 
+/**
+ Japanese users can enable JCB, Diners Club and Discover by calling this method with `YES`, after they have been approved by JCB.
+ 
+ The default value is NO.
+ */
++ (void)setJCBPaymentNetworkSupported:(BOOL)isSupported;
```

Comment:
WDYT about using a class property instead? I think it gives better interop with Swift, but I'm not positive: https://useyourloaf.com/blog/objective-c-class-properties/